templates/account/recipes.html

```diff
@@ -1,3 +1,4 @@
+<!-- recipes page -->
 <div>
-    <div id="main-content-title">My Recipes</div>
+    <div id="main-content-title"><b>My Recipes</b></div>
```

Comment:
I had set up the Profile route to display the user recipes. This must be consolidated after merge